smartcosmos-framework/src/main/java/net/smartcosmos/aspects/SmartCosmosServiceAspect.java

```diff
@@ -14,8 +16,8 @@
 /**
  * Aspect to be applied to extension services implemented in the SMART COSMOS Objects architecture.
  * <p>
- * Using this aspect will capture all exceptions and convert them into a {@see SmartCosmosException}.  If your methods do not declare
- * they throw SmartCosmosException then you will receive an {@see UndeclaredThrowableException}
+ * Using this aspect will capture all exceptions and convert them into a {@link SmartCosmosException}.  If your methods do not declare
+ * they throw SmartCosmosException then you will receive an {@link UndeclaredThrowableException}
```

Comment:
I like `@link` because it directly leads to the related class and its documentation.